File: app/src/main/java/org/fedorahosted/freeotp/external/DropboxClient.java
package org.fedorahosted.freeotp.external;

import com.dropbox.core.DbxRequestConfig;
import com.dropbox.core.v2.DbxClientV2;

/**
 * Created by jezz on 21/06/16.
 */
public class DropboxClient {

    public static DbxClientV2 getClient(String ACCESS_TOKEN) {
        DbxRequestConfig config = new DbxRequestConfig("dropbox/freeopt-cloud");
        DbxClientV2 client = new DbxClientV2(config, ACCESS_TOKEN);
        return client;
    }
}

File: app/src/main/java/org/juanjcsr/newcloudotp/external/DropboxClient.kt
package org.juanjcsr.newcloudotp.external

import com.dropbox.core.DbxRequestConfig
import com.dropbox.core.v2.DbxClientV2

/**
 * Created by jezz on 21/06/16.
 */
object DropboxClient {

    fun getClient(ACCESS_TOKEN: String): DbxClientV2 {
        val config = DbxRequestConfig("dropbox/freeopt-cloud")
        return DbxClientV2(config, ACCESS_TOKEN)
    }
}
